Swashbuckle.AWSApiGateway.Annotations/XAmazonApiGatewayOperationFilter.cs

```diff
@@ -67,9 +67,10 @@ XAmazonApiGatewayIntegrationOptions CreateDefaultIntegrationOptions(OperationFil
                 return new XAmazonApiGatewayIntegrationOptions
                 {
                     HttpMethod = ctx.ApiDescription.HttpMethod,
-                    Uri = Uri.IsWellFormedUriString(baseUri, UriKind.RelativeOrAbsolute)
+                    Uri = _options.IntegrationOptions.Uri == null ? Uri.IsWellFormedUriString(baseUri, UriKind.RelativeOrAbsolute)
                             ? new Uri(new Uri(baseUri), ctx.ApiDescription.RelativePath).ToString()
-                            : $@"{baseUri}{(baseUri.EndsWith("/") ? string.Empty : "/")}{ctx.ApiDescription.RelativePath}",
+                            : $@"{baseUri}{(baseUri.EndsWith("/") ? string.Empty : "/")}{ctx.ApiDescription.RelativePath}"
+                                : null,

```

Comment:
What do you think of this change?
```csharp
                    Uri = !string.IsNullOrWhiteSpace(_options.IntegrationOptions.Uri) 
                        ? _options.IntegrationOptions.Uri
                        : Uri.IsWellFormedUriString(baseUri, UriKind.RelativeOrAbsolute)
                            ? new Uri(new Uri(baseUri), ctx.ApiDescription.RelativePath).ToString()
                            : $@"{baseUri}{(baseUri?.EndsWith("/") ?? false ? string.Empty : "/")}{ctx.ApiDescription.RelativePath}",
```

I added the BaseUri property to allow for the automatic generation of a Uri if none were specified.  To that end, I like the idea of checking for the presence of a valid Uri first and then building one dynamically if necessary.  The null reference is avoided with the addition of the conditional null operator.